src/links/v1/index.html

```diff
@@ -0,0 +1 @@
+providers.json
```

Comment:
Having a HTML link to a JSON file looks like a QnD hack, not a production system. Wouldn't it be better to either auto-generate a decent HTML from the JSON, or, to make things simpler, to have a static `index.html` page with an explicit `<a href="...">` to the `providers.json`? In this way extensions would not "lie" about the file types.

Reply:
Most of these links are workarounds for arranging the hosting of these files from GitHub pages. This is so we do not completely break our present hosting of the providers list while transitioning into Netlify, where we'll get correct content-types, etc. When that transition is done, we can remove them. GitHub pages does not do content negotiation. However, a file or symlink named 'links.html' is also served under 'links', which is a trick to mimic an endpoint.

All that said, that one specific index.html link is a leftover and can be removed. I'll do that.